pyccel/parser/utilities.py

```diff
@@ -5,21 +5,35 @@
 #------------------------------------------------------------------------------------------#
 
 """This file contains different utilities for the Parser."""
+import os
+import random
+import string
 
 from sympy import srepr
-import os
 
 from pyccel.ast.variable       import DottedName
 from pyccel.parser.extend_tree import CommentLine
 from pyccel.ast.internals      import PyccelSymbol
 
+__all__ = ('pyccel_external_lib', 'random_string', 'read_file', 'is_valid_filename_py',
+            'is_valid_filename_pyh', 'header_statement', 'accelerator_statement',
+            'get_module_name', 'view_tree', 'get_default_path')
+
 pyccel_external_lib = {"mpi4py"             : "pyccel.stdlib.external.mpi4py",
                        "scipy.linalg.lapack": "pyccel.stdlib.external.lapack",
                        "scipy.linalg.blas"  : "pyccel.stdlib.external.blas",
                        "scipy.fftpack"      : "pyccel.stdlib.external.dfftpack",
                        "fitpack"            : "pyccel.stdlib.internal.fitpack",
                        "scipy.interpolate._fitpack":"pyccel.stdlib.external.fitpack"}
 
+#==============================================================================
+random_selector = random.SystemRandom()
+
+def random_string( n ):
```

Comment:
Would this function make more sense in `pyccel/utilities/strings.py`? You use it in `epyccel` so it's utility doesn't seem to be limited to the parser

Reply:
yes, it makes sense to put it there , I will change it

Reply:
fixed in [286f989](https://github.com/pyccel/pyccel/pull/1728/commits/286f989446de47a59e9c91a5bc23396be52fa7a2)